test/unit/cookiejar.test.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+process.env.NODE_DEBUG = 'axios';
+jest.mock('axios');
+const axios = require('axios');
+const mockAxiosInstance = jest.fn();
+const tough = require('tough-cookie');
+mockAxiosInstance.defaults = {};
+mockAxiosInstance.interceptors = {
+  request: {
+    use: jest.fn(),
+  },
+  response: {
+    use: jest.fn(),
+  },
+};
+axios.default.create.mockReturnValue(mockAxiosInstance);
+
+const { RequestWrapper } = require('../../dist/lib/request-wrapper');
+const requestWrapperInstance = new RequestWrapper({ jar: new tough.CookieJar() });
+
+describe('axios', () => {
```

Comment:
Should we have a test or two that exercises cookie support?  i.e. we invoke a request, the "endpoint" returns a cookie of some sort in the response, and we then invoke another request and then verify that the cookie is in fact received by the "endpoint" when it receives the second request?